README.md

```diff
@@ -9,57 +9,25 @@
 
 **XMTP client SDK for JavaScript applications**
 
-`xmtp-js` provides a TypeScript implementation of an XMTP client for use with JavaScript and React applications.
+`xmtp-js` provides a TypeScript implementation of an XMTP message API client for use with JavaScript and React applications.
```

Comment:
I'm not sure about the term "XMTP message API client". The word API is throwing me off. In a system as decentralized and E2EE as ours, an "API client" kinda under-sells what this library does. Maybe 5% of the SDK is the API client, and the other 95% is dealing with the protocol as a whole.

Reply:
Thank you very much for this, @neekolas - I'll happily revert to "XMTP client"!